ui/tests/integration/components/sync/secrets/page/destinations/create-and-edit-test.js

```diff
@@ -212,6 +212,16 @@ module('Integration | Component | sync | Secrets::Page::Destinations::CreateAndE
     gh: ['accessToken'],
     'vercel-project': ['accessToken', 'teamId', 'deploymentEnvironments'],
   };
+  const EXPECTED_VALUE = (key) => {
```

Comment:
pulled into method so can be extended in the future for other destination types/attrs